src/inputs/BasicInputs/BasicSliderInput/BasicSliderInput.js

```diff
@@ -57,7 +57,7 @@ export const BasicSliderInput = (props) => {
         <Slider
           value={getValue(value)}
           id="slider-root"
-          data-cy="slider-root"
+          data-cy="slider-input"
```

Comment:
In fact, I made change for _data-cy_ and _id_ to _slider-root_ to avoid confusion with real 'input' child element.

Reply:
The whole point of `data-cy` is to be more constant than ids and classes to prevent breaking tests too often, so I thought we could keep the `slider-input` value for `data-cy` :slightly_smiling_face: (and it's still consistent with the component name `BasicSliderInput`)
If we change this one, then we should also change `data-cy="slider-input-textField"` to be consistent, and break even more tests :smile: